lib/ex_dtls.ex

```diff
@@ -60,6 +66,36 @@ defmodule ExDTLS do
     GenServer.start_link(__MODULE__, opts)
   end
 
+  @doc """
+  Generates new certificate.
+
+  Returns DER representation in binary format.
+  """
+  @spec generate_cert(pid :: pid()) :: cert :: binary()
+  def generate_cert(pid) do
+    GenServer.call(pid, :generate_cert)
+  end
+
+  @doc """
+  Gets current private key.
+
+  Returns key specific representation in binary format.
+  """
+  @spec get_pkey(pid :: pid()) :: pkey :: binary()
+  def get_pkey(pid) do
+    GenServer.call(pid, :get_pkey)
+  end
```

Comment:
I'm wondering if this could automatically start and kill the CNode

Reply:
I am not sure if adding additional logic here is a good idea. `ExDTLS` is written as CNode so it's reasonable to start CNode at first. Also we might want to get pkey/cert of an existing ExDTLS instance e.g. for decrypting packets in Wireshark (in the future).

Reply:
Ok, so I'd add some `stop` function so that one doesn't have to kill the process manually